.github/workflows/audit.yml

```diff
@@ -22,4 +22,4 @@ jobs:
       - uses: pnpm/action-setup@v4
         with:
           version: 9.4
-      - run: pnpm audit
+      - run: pnpm audit --prod
```

Comment:
@h0ngcha0 don't you think it makes sense to only audit packages that are used in production

https://pnpm.io/cli/audit#--prod--p

Reply:
Sorry for being late, would it be an issue if we audit all packages? `devDependencies` might affect our dev environment so if we can maybe we should audit as well, WDYT?

Reply:
ok, fair enough!